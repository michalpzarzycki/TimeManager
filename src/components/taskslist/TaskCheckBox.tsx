import React from 'react';
import styles from './TaskCheckBox.module.css'
export default function TaskCheckBox({ task, handleChecked }: any) {
    return (
    <div className={styles.checkbox}>
        <input type="checkbox" checked={task.done} name="taskCheckbox" onClick={() => handleChecked(task)} />
    </div>)

}