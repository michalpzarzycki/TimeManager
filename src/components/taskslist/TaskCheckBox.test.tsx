import React from 'react'
import {shallow} from 'enzyme'
import TaskCheckBox from './TaskCheckBox'

test('Expext to render TaskCheckBox component', () => {
    expect(shallow(<TaskCheckBox task={{}} handleChecked={() => {}}/>)).toMatchSnapshot()
})
