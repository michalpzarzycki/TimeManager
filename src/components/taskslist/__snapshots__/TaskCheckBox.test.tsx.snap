// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expext to render TaskCheckBox component 1`] = `ShallowWrapper {}`;
